src/databricks/labs/ucx/queries/progress/01_0_percentage_migration_readiness.sql

```diff
@@ -0,0 +1,5 @@
+/* --title 'Overall readiness (%)' --width 2 */
+SELECT
+    100 * COUNT_IF(size(failures) = 0) / COUNT(*) AS percentage
+FROM multiworkspace.historical
+WHERE object_type != 'migration_status'
```

Comment:
I'm a bit on the fence on this one; part of me wonders whether we should list the known object-types (to deal with another version introducing something like `migration_status` that should be excluded).

Linting issues are also coming shortly: these are per-problem records instead of per-resource.

Reply:
I made the check inclusive instead of exclusive